mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -218,24 +222,44 @@ class SetpointRawPlugin : public plugin::PluginBase,
 	{
 		Eigen::Quaterniond desired_orientation;
 		Eigen::Vector3d baselink_angular_rate;
-
+		Eigen::Vector3d body_rate;
+		double thrust;
+		// Set Thrust scaling in px4_config.yaml, setpoint_raw block.
```

Comment:
Neet blank line between definition block and following code.

Reply:
I don't really know how this came here